sigstore/oidc.py

```diff
@@ -247,7 +246,10 @@ def staging(cls) -> Issuer:
         return cls(STAGING_OAUTH_ISSUER_URL)
 
     def identity_token(  # nosec: B107
-        self, client_id: str = "sigstore", client_secret: str = ""
+        self,
+        client_id: str = "sigstore",
+        client_secret: str = "",
+        force_oob: bool = False,
     ) -> IdentityToken:
         """
         Retrieves and returns an `IdentityToken` from the current `Issuer`, via OAuth.
```

Comment:
Let's update the docstring here as well, to emphasize that `force_oob` modifies the browser-opening behavior.

Reply:
PTAL

Reply:
Looks good, thank you!